keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/DeliveryArtifact.kt

```diff
@@ -1,53 +1,36 @@
 package com.netflix.spinnaker.keel.api.artifacts
 
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.BRANCH_JOB_COMMIT_BY_JOB
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.SEMVER_JOB_COMMIT_BY_JOB
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.SEMVER_JOB_COMMIT_BY_SEMVER
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.SEMVER_TAG
-
 enum class ArtifactType {
-  deb, docker
+  deb, docker;
+
+  override fun toString() = name
 }
 
 enum class ArtifactStatus {
   FINAL, CANDIDATE, SNAPSHOT, RELEASE, UNKNOWN
 }
 
-sealed class DeliveryArtifact {
+/**
```

Comment:
Why did you move these things to the `DeliveryArtifact` class? I feel like if you're going to move them they might as well be moved into their own classes.

Reply:
Fair point. Will do.